### ISSUE TITLE
package.json lists missing file, confuses eslint-plugin-imports

### ISSUE THREAD

**Issue by ajslater** [huynamboz/vue-drag-scroller#40]
Thank you for this package! I like how small and efficient it is!

the package.json lists 

```js
"main": "dist/vue-drag-scroller.js",
...
      "require": "./dist/vue-drag-scroller.js",
```
these files, which don't exist. I think this is what's confusing the eslint-plugin-imports package. Which throws linting errors at me.

There is an easy workaround for users of this plugin: Just import the actual file:

```js
import VueDragScroller from "vue-drag-scroller/dist/vue-drag-scroller.es.js";  
```

Proposed fixes for this package might be:
1) Rename `vue-drag-scroller.es.js` to `vue-drag-scroller.js` and fix the package.json
Or
2) Fix the package.json main and require blocks to use `dist/vue-drag-scroller.es.js`
Or
3) Include a symlink in the package pointing from `vue-drag-scroller.es.js` to `vue-drag-scroller.js`

**Comment by huynamboz** [comment]
thank for issue, i will enhance it, respect bro @ajslater

**Comment by huynamboz** [comment]
Hi, i fixed it, you can upgrade package to `v1.8.6` @ajslater
